code/modules/surgery/organs/internal/lungs/_lungs.dm

```diff
@@ -937,6 +952,14 @@
 	heat_level_2_threshold = 473
 	heat_level_3_threshold = 1073
 
+/obj/item/organ/internal/lungs/ethereal/ethereal_smoker
+	name = "smoker aeration reticulum"
+	desc = "A pair of exotic lungs that look pale and sickly, a result from smoking a lot."
+	icon_state = "lungs_ethereal-s"
+
+	maxHealth = STANDARD_ORGAN_THRESHOLD * 0.75
+	healing_factor = STANDARD_ORGAN_HEALING * 0.75
```

Comment:
Rather than paste the same calculations six times it would make sense to have (as an example)
```
#define SMOKER_LUNG HEALING (STANDARD_ORGAN_HEALING * 0.75)

/obj/item/organ/internal/lungs/ethereal/ethereal_smoker
	healing_factor = SMOKER_LUNG_HEALING
```